containers/Investment/View/index.tsx

```diff
@@ -57,10 +57,9 @@ class InvestmentsView extends React.Component<Props, State> {
         </Box>}
 
       {analytics && analytics.data && auth && auth.user && auth.user.permissions.canSetMinimumJuniorRatio &&
-        <JuniorRatio tinlake={tinlake} minJuniorRatio={analytics.data.minJuniorRatio}> </JuniorRatio>
+        <JuniorRatio tinlake={tinlake} minJuniorRatio={analytics.minJuniorRatio} />
```

Comment:
minJuniorRatio has to be part of data. This is how the data object looks like: 
`data: {
        junior: {
          type: "junior",
          availableFunds: juniorReserve,
          tokenPrice: juniorTokenPrice,
          token: "TIN"
        },
        senior: {
          type: "senior",
          availableFunds: seniorReserve,
          tokenPrice: seniorTokenPrice,
          token: "DROP",
          interestRate: seniorInterestRate
        },
        availableFunds: juniorReserve.add(seniorReserve),
        minJuniorRatio: minJuniorRatio,
        currentJuniorRatio: currentJuniorRatio
      }
    }`

There is an inital mistake in the AuthState type:
`export interface AnalyticsState {
  state: null | 'loading' | 'found';
  data: null | {
    junior: Tranche,
    senior?: Tranche
  };
  availableFunds: BN,
  minJuniorRatio: BN,
  currentJuniorRatio: BN
}`

  availableFunds, minJuniorRatio, currentJuniorRatio should be part of data

Reply:
Good finding, I'll change that.